lxd/storage/backend_lxd.go

```diff
@@ -6320,6 +6328,12 @@ func (b *lxdBackend) UpdateCustomVolume(projectName string, volName string, newD
 		})
 	}
 
+	// Update the instance's backup files.
```

Comment:
You even used "s" in the comment here.

Thats a +1 for `UpdateCustomVolumeBackupFiles` right? ;)